datadog/resource_datadog_synthetics_test_.go

```diff
@@ -272,7 +273,11 @@ func newSyntheticsTestFromLocalState(d *schema.ResourceData) *datadog.Synthetics
 		options.SetTickEvery(tickEvery)
 	}
 	if attr, ok := d.GetOk("options.follow_redirects"); ok {
-		followRedirects := attr.(string) == "1"
+		// TF nested schemas is limited to string values only
+		// follow_redirects being a boolean in Datadog json api
+		// we need a sane way to convert from boolean to string
+		// and from string to boolean
```

Comment:
Not sure of the value of this comment

Reply:
Comment provide context to explain why we convert to "true" / "false" string: prevents breaking code in the future + give insights if we ever need to refacto